### ISSUE TITLE
[Adaptive Card iOS - ColumnSet.Input.ChoiceSet.VerticalStretch]: Voice over is not conveying the change of state when user is deselecting the checkboxes.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8433]
### Target Platforms

iOS

### SDK Version

1.0 (2.4.0-beta.23.1.8.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 16.3.1
Application: Adaptive Cards
App Version: 1.0 (2.4.0-beta.23.1.8.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.1" control and select it.
5. Navigate to 'Test' control and activate it.
6. Navigate to 'ColumnSet.Input.ChoiceSet.VerticalStretch.Jsons' and activate it.
7. Navigate to any checkboxes and double tap on it and observe the issue.

**Actual Result:**
After deselecting any checkboxes, voice over is not conveying the change of state for the same. It is announced as **'Comma green comma checkbox'**. Also, it is announcing the incorrect information as double tap to selected for already selected checkbox.

**Expected Result:**
Voice over should convey the change of state when user deselect any checkboxes. and it should announce as **'Green checkbox not selected'**. Also, it should not announce the incorrect information as 'Double tap to select for the already selected- checkbox.'

**Observation:**
While renavigating to the same control, voice over is conveying the state information correctly.

**User Impact:**
It will impact the screen reader as they will not able to know whether action has been performed or not?

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/95913748/228840249-6ed7e1f3-8bbe-4165-b3de-3079cf7bb53c.MP4

### Card JSON

```JSON
NA
```

**Comment by jwoo-msft** [comment]
closing this issue, the app wasn't built correctly perhaps, only cards under Scenario should be tested, not under Elements. I will make note of this and fix it.
